extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -214,7 +214,7 @@ export const getBlockedDomains = createAsyncThunk<
   }
 });
 
-export enum HwOverlayStatus {
+export enum ShowOverlayStatus {
```

Comment:
changed this to something more general that we can re-use anytime we need a screen overlay, eg. ledger signing or this warning

Reply:
if there's a better word than "overlay" that describes what this is doing happy to change the name 😄